### PR TITLE
Add a tab showing status of all workflows

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

File: dashboard.Rmd [hunk 1]
```diff
@@ -338,6 +338,31 @@ reactable(
...
+    correlation = colDef(html = TRUE),
+    datawizard = colDef(html = TRUE),
+    easystats = colDef(html = TRUE),
+    effectsize = colDef(html = TRUE),
+    insight = colDef(html = TRUE),
+    modelbased = colDef(html = TRUE),
+    parameters = colDef(html = TRUE),
+    performance = colDef(html = TRUE),
+    report = colDef(html = TRUE),
+    see = colDef(html = TRUE)
```
Comment (IndrajeetPatil) [inline dashboard.Rmd:360]: This is hideous. Is there a better way to do this in `{reactable}`?

Comment (IndrajeetPatil) [reply dashboard.Rmd:360]: Oh, I can use `defaultColDef` argument.

**Comment by IndrajeetPatil** [comment]
@etiennebacher Can you please check locally how this looks and see if its look can be improved? 

I am not happy with how cramped the table looks. It will be better either to display smaller SVG badges or let the table scroll horizontally past 100%vw. But I haven't yet looked into how to do either in _reactable_.

File: data-raw/workflow_badge_urls_db.R [hunk 2]
```diff
@@ -0,0 +1,51 @@
+extract_gha_workflows <- function(username,
+                                  repository,
+                                  folder_path = ".github/workflows",
+                                  token = Sys.getenv("GITHUB_PAT")) {
```
Comment (IndrajeetPatil) [inline data-raw/workflow_badge_urls_db.R:4]: What's the token we are using for this repo? I guess that's what we will need here?

**Comment by etiennebacher** [comment]
> I am not happy with how cramped the table looks. It will be better either to display smaller SVG badges or let the table scroll horizontally past 100%vw. But I haven't yet looked into how to do either in reactable.

Some check names are particularly long (e.g `R-CMD-check-devel-easystats` so rescaling the SVG badges will make them super small (also I don't know how hard it is to do). Having a larger scrollable table is also not ideal IMO.

Since the check name is displayed in the first column, we don't really need the badges themselves. Is it possible to retrieve the status of each workflow (success/failure) via the API? If so, we could just replace the badges by a "Passed" or "Failed" label that links to the check.

File: data-raw/workflow_badge_urls_db.R (hunk 2 above)
Comment (etiennebacher) [reply data-raw/workflow_badge_urls_db.R:4]: It is now one of Rémi's tokens, it is automatically retrieved in the action workflow.

**Comment by IndrajeetPatil** [comment]
@etiennebacher Looks much better, no? 😄 

Screenshot 2023-09-29 at 12 15 20

**Comment by etiennebacher** [comment]
Yeah, looks really nice, thanks!

**Comment by IndrajeetPatil** [comment]
@etiennebacher Since this PR was merged, the dashboard hasn't rendered successfully on the main branch. I don't understand why it would succeed in the PR but not in the default branch.

The error we get seem to stem from us hitting the GitHub API a lot, but we are using stored `.rds` data containing URL strings to create check emojis, so we shouldn't be hitting the API too much. Not sure what's going on.

**Comment by etiennebacher** [comment]
I don't get it, I see several successes in GHA and the date displayed on the dashboard is today. There are always some failures from time to time

IMG_20231002_181759

**Comment by IndrajeetPatil** [comment]
Strangely, our workflow builds and deploys the dashboard even when triggered for a PR. This is not desirable but it's also not a top priority issue to fix at the moment. The successes you are are from my draft PR.

**Pull request merged**